pkg/activator/net/throttler.go

```diff
@@ -49,18 +49,23 @@ import (
 )
 
 const (
-
 	// The number of requests that are queued on the breaker before the 503s are sent.
 	// The value must be adjusted depending on the actual production requirements.
 	breakerQueueDepth = 10000
 
 	// The upper bound for concurrent requests sent to the revision.
-	// As new endpoints show up, the Breakers concurrency increases up to this value.
-	breakerMaxConcurrency = 1000
+	// As new endpoints show up, the revisionThrottler's concurrency increases up
+	// to this value.  We need to set some value here since the breaker requires
+	// an explicit buffer size (it's backed by a chan struct{}), but it can
+	// safely be quite large.
+	breakerMaxConcurrency = 100000
```

Comment:
Any reason this is not just the max buffer length of channels anyway? Or Int32Max or something?

Maybe we should set a constant in the breaker implementation itself and reuse that here. I.e. `queue.MaxBreakerCapacity` and describe there why it's set to the respective value.

Reply:
yeah, can't think of a reason not to, and the constant in queue is a good idea. Turned out this needed some test fixes since they try to create MaxConcurrency trackers, though.